beast/observationmodel/noisemodel/toothpick.py

```diff
@@ -176,7 +188,7 @@ def _compute_sigma_bins(
         good_bins = np.zeros(nbins, dtype=int)
 
         # get the indexs to the recovered fluxes
-        good_indxs, = np.where(flux_out != 0.0)
+        (good_indxs,) = np.where(flux_out != 0.0)
```

Comment:
Do we want to use math.isclose for comparing floats?

Reply:
In this case, a flux of precisely zero is what indicates a source wasn't recovered.  I don't know if there's ever a chance of floating point errors messing that up, but I would suspect not, since `flux_out` is read directly from the fits file.

Reply:
Why the parenthesis on the left-hand side? my editor says this is not necessary. Is this a python 2.x thing? This may be better to use np.where(...)[0]??

Reply:
This is inserted by `black` when I run it.  Been using this to auto-format code to avoid having to remember all the bloody PEP8 stuff.  It is just an explicit way to say it is a tuple.  There is differences of opinion in the community.  `black` picks a solution and avoids us having to decide.

Reply:
For more info: https://github.com/psf/black/issues/1108